learning.py

```diff
@@ -739,7 +739,8 @@ def BackPropagationLearner(dataset, net, learning_rate, epochs, activation=sigmo
 
             # Initialize delta
             delta = [[] for _ in range(n_layers)]
-
+            if momentum == True:
+            	v_dw = [[0 for i in range(len(_))] for _ in net]
```

Comment:
I think the indentation should be longer, although it may be a glitch in my browser.